### DISCUSSION TITLE
Need to restore a deleted team

### DISCUSSION THREAD

**Discussion by stlln** [vercel/community#4120] [Account Issue]
### Question

Is that possible?

[This post](https://vercel.com/blog/changelog-march-2020) mentions that it can be recovered for up to 7 days

**Comment by amyegan** [comment]
Hi @stlln. Sorry, that post is a dated from a few years ago and is no longer accurate regarding team deletion. When attempting to delete a team, you are presented with some warnings and confirmation checks. If I try to delete my team right now, I see this confirmation box that says `Warning: This action is not reversible.`

<img width="513" alt="Account settings warning that account deletion is permanent" src="https://github.com/vercel/community/assets/8247105/72c90651-ddb6-42ef-b3e6-8dc7c9ece6d1">
